src/Core/src/Platform/Windows/NavigationViewExtensions.cs

```diff
@@ -10,6 +10,31 @@ namespace Microsoft.Maui.Platform
 {
 	public static class NavigationViewExtensions
 	{
+		// This is mainly used by UnitTest to grab the elements and test values.
+		// At one point I was using this inside the application but it started to cause erratic behavior
```

Comment:
Is this not used at all in the application anymore? If so, should it be an extension method in the unit test projects, rather than living here?

Reply:
👍 moved